mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -454,15 +454,19 @@ export const WayPage = (props: WayPageProps) => {
               onClick={() => updateWayPageSettings({isJobDoneTagsVisible: !wayPageSettings.isJobDoneTagsVisible})}
             >
               {wayPageSettings.isJobDoneTagsVisible ?
-                <Icon
-                  size={IconSize.MEDIUM}
-                  name="EyeOpenedIcon"
-                />
+                <button className={styles.iconWrapper}>
+                  <Icon
+                    size={IconSize.MEDIUM}
+                    name="EyeOpenedIcon"
+                  />
+                </button>
                 :
```

Comment:
Maybe will be better to use one button and change name depends on wayPageSettings.isJobDoneTagsVisible instead of render 2 buttons?

Reply:
fixed